tests/testsuite/auto_complete/common.rs

```diff
@@ -0,0 +1,68 @@
+pub(crate) fn complete(input: &str, shell: impl Into<Shell>) -> String {
+    let shell = shell.into();
+
+    // Load the runtime
+    let (mut runtime, _scratch) = match shell {
+        Shell::Bash => load_runtime::<completest_pty::BashRuntimeBuilder>(),
+        Shell::Zsh => load_runtime::<completest_pty::ZshRuntimeBuilder>(),
+    };
+
```

Comment:
This feels roundabout to go from `&str` -> `Shell` -> `completest_pty::*RuntimeBuilder`